javaslang/src/main/java/javaslang/collection/Vector.java

```diff
@@ -690,7 +687,9 @@ public T get(int index) {
         if ((index < 0) || (index >= length())) {
```

Comment:
Would it be ok if I rather pushed an updated formatter config instead, that supports aligning ternaries?
```File | Settings | Editor | Code Style | Java | Wrapping and Braces | Align when multiline ☑```